### PR TITLE
Hook up the android kotlin code to the send js code

### PR DESCRIPTION
**Pull request by fzzzy**
- Generate a bundle for android from android/android.js
- Temporarily remove -p from webpack since uglify chokes on things in android.js for some reason
- Encryption runs properly but then things go wrong when api.js tries to post the file to the server since the origin is null and the protocol is file

### CODE REVIEW

File: android/android.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
+/* global window, document, fetch */
+
+const MAXFILESIZE = 1024 * 1024 * 1024 * 2;
+
+const EventEmitter = require('events');
+const emitter = new EventEmitter();
+
+function dom(tagName, attributes, children = []) {
```
Comment (dannycoates) [inline android/android.js]: `choo/html` does pretty much exactly this. Is there a reason to not use it here?

File: app/api.js [hunk 2]
```diff
@@ -137,7 +137,12 @@ async function upload(
   const host = window.location.hostname;
   const port = window.location.port;
   const protocol = window.location.protocol === 'https:' ? 'wss:' : 'ws:';
-  const ws = await asyncInitWebSocket(`${protocol}//${host}:${port}/api/ws`);
+  const endpoint =
+    window.location.protocol === 'file:'
+      ? 'wss://send2.dev.lcip.org/api/ws'
```
Comment (dannycoates) [inline app/api.js]: How should we do this for real? A build-time variable in webpack maybe?

File: app/fileManager.js [hunk 3]
```diff
@@ -108,8 +108,10 @@ export default function(state, emitter) {
       if (cancelBtn) {
         cancelBtn.hidden = 'hidden';
       }
-      await delay(1000);
-      await fadeOut('.page');
+      if (document.querySelector('.page')) {
```
Comment (dannycoates) [inline app/fileManager.js]: 👍

File: app/api.js (hunk 2 above)
Comment (fzzzy) [reply app/api.js]: Good question. I created an issue to track this. https://github.com/mozilla/send/issues/869

We should do it however other similar configuration of this type is handled. A build-time variable in webpack seems like the right thing. Are there other things configured in this way already?

**Pull request merged**